src/main/java/com/stuypulse/robot/commands/conveyor/ConveyorToAmp.java

```diff
@@ -26,6 +26,7 @@ public ConveyorToAmp() {
     public void execute() {
         conveyor.toAmp();
         intake.acquire();
+        amper.score();
```

Comment:
Shouldn't this call `amper.intake()` instead?

Reply:
I just reread what `amper.intake()` does. It doesn't need to exist? We're never going to use it.

Reply:
I think we leave it, maybe delete amperintake command